inductor/src/main/java/com/oneops/inductor/WorkOrderExecutor.java

```diff
@@ -275,52 +279,67 @@ public String getKitchenTestPath(CmsWorkOrderSimple wo) {
             responseMap.put("task_result_code", "500");
             return responseMap;
           }
-          String destDir = "/tmp/" + getShortenedClass(wo.getRfcCi().getCiClassName()) + "-" + wo
-              .getDeploymentId();
-          File dest = new File(destDir);
-          FileSystemUtils.deleteRecursively(dest);
-          FileSystemUtils.copyRecursively(new File(getKitchenTestPath(wo)), dest);
-          String kitchenConfig = generateKitchenConfig(wo, sshKeyPath, logKey);
-          String kitchenConfigPath = format("%s/%dk.yaml", destDir, wo.getDpmtRecordId());
-
-          Files.write(Paths.get(kitchenConfigPath), kitchenConfig.getBytes(),
-              StandardOpenOption.CREATE_NEW);
-
-          logger
-              .info(logKey + "Generated kitchen config for deployment id: " + wo.getDpmtRecordId());
-          logger.info(logKey + "Kitchen Config Path: " + kitchenConfigPath);
-          logger.info(logKey + "SSH key path: " + sshKeyPath);
-          logger.info(logKey + "Remote WO Path: " + remoteWOPath);
-          logger.info(logKey + "CookbookPath: " + cookbookPath);
-          logger.info(logKey + "Working Dir: " + destDir);
-
-          //Execute the kitchen verify
-          Map<String, String> envVars = new HashMap<>();
+        }
+        String destDir = "/tmp/" + getShortenedClass(wo.getRfcCi().getCiClassName()) + "-" + wo
+            .getDeploymentId();
+        File dest = new File(destDir);
+        FileSystemUtils.deleteRecursively(dest);
+        FileSystemUtils.copyRecursively(new File(getKitchenTestPath(wo)), dest);
+        if (isRemoteChefCall(wo)) {
+          kitchenConfig = generateKitchenConfig(wo, sshKeyPath, logKey);
+        } else {
```

Comment:
We don't need this check since we are passing `sshKeyPath`, which is null for local wo.

Reply:
Apart from that LGTM.